grammars/php.cson

```diff
@@ -1299,7 +1306,7 @@
       }
       {
         # Root namespace function calls (built-in or user)
-        'begin': '(?i)(\\\\)?\\b([a-z_\\x{7f}-\\x{ff}][a-z0-9_\\x{7f}-\\x{ff}]*)\\s*(\\()'
+        'begin': '(?i)(\\\\)?\\b([a-z_\\x{7f}-\\x{7fffffff}][a-z0-9_\\x{7f}-\\x{7fffffff}]*)\\s*(\\()'
```

Comment:
Same as above, `(?<![a-z0-9_\\x{7f}-\\x{7fffffff}])` should be a good replacement.